docs/public/content/admin/index.md

```diff
@@ -0,0 +1,6 @@
+# DevOps
+
+@@PROJECT_NAME@@ is developed by SAP for internal use. No service is provided to Internet users and no instance is available on the Internet. Anyway, we believe that @@PROJECT_NAME@@ should be usable by everyone. We thus made tutorials to deploy @@PROJECT_NAME@@ easily with Docker.
```

Comment:
@@PROJECT_NAME@@ is meant to run within the internal network of software development organizations, and this tutorial is meant to facilitate the operation of @@PROJECT_NAME@@ using **local** Docker registries or **local** container orchestration systems (e.g., Kubernetes or OpenShift).

**Important**: Due to the current lack of an authorization concept, we do not advise to expose an instance of @@PROJECT_NAME@@ to the Internet.